### ISSUE TITLE
UI Sizing Issues for certain Players

### ISSUE THREAD

**Issue by cecilia-sanare** [butter-cookie-kitkat/ButterQuestTracker#51]
## Bug description

Some users are experience inputs overflowing to the next line or text getting truncated, need to figure out the cause.

_Images courtesy of @Nillx_

## Screenshots

![Filters & Sorting](https://i.imgur.com/n4lATJX.jpg)

![Visual Settings](https://i.imgur.com/jb3Zr6y.jpg)

![Frame Settings](https://i.imgur.com/NAIqcSC.jpg)

![Advanced](https://i.imgur.com/dJIhLGP.jpg)

**Butter Quest Tracker Version:** 1.9.0-rc.2

**Comment by cecilia-sanare** [comment]
@Nillx, can you post an updated screenshot of the "Filters & Sorting" page with 1.9.1?

**Comment by Nillx** [comment]
https://i.imgur.com/CBdYV6n.jpg

**Comment by cecilia-sanare** [comment]
Thanks for the quick response! I'll have to try and reproduce the sizing issues on my screen.

**Comment by Nillx** [comment]
Sizing issue is where the text is cut and have ... ?

**Comment by cecilia-sanare** [comment]
That and the inputs get wrapped onto their own line.

As an example "Automatically Track Updated Quests" and "Sorting" should be on a single line.

**Comment by Nillx** [comment]
I see them on the same line when i turn off Zygor guides, text is still cut tho.

https://imgur.com/a/KjTcGxz

Text is getting cut off even without addons

**Comment by cecilia-sanare** [comment]
Interesting, thanks for taking the time to look into it!

So disabling every mod, but Butter Quest Tracker still results in the text getting truncated?
If that's the case my best guess would be that it's either Resolution or DPI related.

**Comment by Nillx** [comment]
Yes. Text was truncated even without any addons. I will play around with the resolution/UI scaling tomorrow when i wake up.

**Comment by Nillx** [comment]
1.9.2 fixed the text issue. Also i found out that Zygor have an option to resize the Interface options, once i made the menu a little bit wider, the text moved on a single bar as it should be.

**Comment by Nillx** [comment]
Edit: Its not fixed, after reloading the text got cut again. I did found what is causing it atleast, its the UI scaling. Here are my UI scaling settings, first ones are the one i use (text getting truncated) and on the next one its fixed but the UI is slightly bigger. https://imgur.com/a/uTMWlJz
